lms/static/file-icon-vivid.min.css

```diff
@@ -0,0 +1,1118 @@
+/*!
```

Comment:
Thtat's called `min` but it doesn't look like a minified version

Reply:
As you can see in the documentation I downloaded it from here: https://fileicons.org/
I can change the file name if you want to `file_icons.css` or something like that.